src/Components/Members/index.jsx

```diff
@@ -0,0 +1,120 @@
+import { useEffect, useState } from 'react';
+import styles from './members.module.css';
+import Table from './Table';
+import Form from './Form';
+
+const Members = () => {
+  const [members, setMembers] = useState([]);
+  const [memberValues, setMemberValues] = useState({
+    firstName: '',
+    lastName: '',
+    email: '',
+    dni: '',
+    phone: '',
+    city: '',
+    birthday: '2023-05-29T07:32:26+0000',
+    postalCode: '',
+    membership: '',
+    isActive: true
+  });
+  const [idMember, setIdMember] = useState('');
+  const getMembers = async () => {
+    const response = await fetch(`${process.env.REACT_APP_API_URL}/members`);
+    const data = await response.json();
+    setMembers(data.data);
+    console.log(data);
+  };
+
+  useEffect(() => {
+    getMembers();
+  }, []);
+
+  const changeDateFormat = (date) => {
+    let dateArray = date.split('-');
+
+    let year = dateArray[0];
+    let month = dateArray[1];
+    let day = dateArray[2];
+
+    dateArray = month + '/' + day + '/' + year;
+    return dateArray;
+  };
+
+  const updateMember = async (idMember) => {
+    const dateFormat = changeDateFormat(memberValues.birthday);
+    try {
+      const response = await fetch(`${process.env.REACT_APP_API_URL}/members/${idMember}`, {
```

Comment:
remember to implement the /api/members fix discussed on slack
<img width="703" alt="Screenshot 2023-05-30 at 23 26 32" src="https://github.com/BaSP-m2023/joaco-megarocket-app/assets/127536320/58737b85-a1c9-45d7-b878-a8769b1d4344">
![fix Image 2023-05-30 at 20 39 22](https://github.com/BaSP-m2023/joaco-megarocket-app/assets/127536320/741cf030-c03b-42fc-8206-6c099034d91c)